### PR TITLE
NDEV-532 json logger for proxy and neon-cli

### PR DESCRIPTION
**Pull request by gigimon**

### CODE REVIEW

File: proxy/common_neon/solana_tx_error_parser.py [hunk 1]
```diff
@@ -2,25 +2,27 @@
...
+
+
 class SolTxError(Exception):
     def __init__(self, receipt: SolTxReceipt):
         self.result = receipt
 
         log_list = SolTxErrorParser(receipt).get_log_list()
-        self.error = '. '.join([log for log in log_list if self._is_program_log(log)])
-        if not len(self.error):
+        LOG.error = '. '.join([log for log in log_list if self._is_program_log(log)])
```
Comment (afalaleev) [inline proxy/common_neon/solana_tx_error_parser.py]: This is an internal parameter of the exception.

File: proxy/common_neon/solana_tx_error_parser.py [hunk 2]
```diff
@@ -2,25 +2,27 @@
...
+        if not len(LOG.error):
```
Comment (afalaleev) [inline proxy/common_neon/solana_tx_error_parser.py]: the same

File: proxy/common_neon/solana_tx_error_parser.py [hunk 3]
```diff
@@ -2,25 +2,27 @@
...
             self.error = json.dumps(receipt)
 
-        super().__init__(self.error)
+        super().__init__(LOG.error)
```
Comment (afalaleev) [inline proxy/common_neon/solana_tx_error_parser.py]: the same

File: proxy/testing/test_get_block_hash.py [hunk 4]
```diff
@@ -82,16 +81,6 @@ def test_getBlockHashFromHistory(self):
         print(logs)
         self.assertEqual(logs[0], block_hash_history)
 
-    def test_get_fake_block_by_hash(self):
```
Comment (afalaleev) [inline proxy/testing/test_get_block_hash.py]: why do you remove this test?

**Review by afalaleev — CHANGES_REQUESTED** [review]
Please fix small problems

File: proxy/testing/test_get_block_hash.py (hunk 4 above)
Comment (gigimon) [reply proxy/testing/test_get_block_hash.py]: return it (can't remember)

**Review by afalaleev — DISMISSED** [review]
Please describe the final result in the commit comment, when you squash the commit.
Don't leave the squashed comments from all included commits.

**Review by afalaleev — APPROVED** [review]

**Pull request merged**